WCON_format.md

```diff
@@ -280,12 +283,10 @@ An example WCON file with a complete metadata section is given below.
            "media":"NGM",
            "sex":"hermaphrodite",
            "stage":"dauer",
-           "age":38.4,
+           "age":18.511,
```

Comment:
Not being an actual researcher, you've caught me out on using unrealistic data.  Great point, I'll revise it back to your number.  I only changed it at all because I first changed it back to a string and then I just made up a number rather than reverting.